server/scripts/droneRoute.js

```diff
@@ -0,0 +1,113 @@
ma","lat":35.7083599,"lng":139.770988},{"id":"ChIJodnuomCSGGARncvT06nj9EA","name":"Oji-Kamiya","lat":35.765424,"lng":139.735626},{"id":"ChIJEzKTRveLGGAR3JwAGCgXTdQ","name":"Nijubashimae","lat":35.680628,"lng":139.76185},{"id":"ChIJ65lhJ-ryGGAR-aILJ5ikFd8","name":"Shin-Nakano","lat":35.6976549,"lng":139.6698209},{"id":"ChIJwR3tffzyGGARdQLxp018iZ0","name":"Honancho","lat":35.68348599999999,"lng":139.657914},{"id":"ChIJr2jeH2TsGGARK5JVeRJcpz4","name":"Chikatetsu Akatsuka","lat":35.769976,"lng":139.644023},{"id":"ChIJLS3M1YzyGGARjajeHAu8GS0","name":"Higashi-Koenji","lat":35.69792899999999,"lng":139.658274},{"id":"ChIJH8bIvpGOGGARbsQ_EEA4hhM","name":"Iriya","lat":35.7210426,"lng":139.7865085},{"id":"ChIJkUe-3yKNGGARaIlMWZpmiHg","name":"Nishi-Waseda","lat":35.7078154,"lng":139.7091474},{"id":"ChIJScLILb-OGGARaG4VCipgC8c","name":"Tawaramachi","lat":35.7098045,"lng":139.7907555},{"id":"ChIJv5f9ejOMGGARQ8SNRZCaUOc","name":"Todai-mae","lat":35.717818,"lng":139.758015},{"id":"ChIJZYDOLt2PGGAR1s4dVGAZPg4","name":"Kita-Ayase","lat":35.776869,"lng":139.832118},{"id":"ChIJYdtNvy2MGGAR_EHnmgMU0X8","name":"Nezu","lat":35.7194307,"lng":139.7641769},{"id":"ChIJtdg4j_aOGGAR-ZZnaku22kU","name":"Minami-Senju","lat":35.7333194,"lng":139.7990561},{"id":"ChIJ7VMs1MqJGGARqwyIqRtH1TE","name":"Tatsumi","lat":35.6436264,"lng":139.8194492},{"id":"ChIJU7L1V86NGGAR1_6tOPSjURg","name":"Sendagi","lat":35.7268018,"lng":139.7581383},{"id":"ChIJxXiWgMmAGGARYHq-JcmyyUw","name":"Baraki-Nakayama","lat":35.7031661,"lng":139.941603},{"id":"ChIJja5N4bX8GGARD9Kmr6bA6dg","name":"Ochiai","lat":35.6096013,"lng":139.4278927},{"id":"ChIJnTEx_Z-OGGARgdDuI__dhuI","name":"Ueno-Hirokoji","lat":35.7079034,"lng":139.7730918},{"id":"ChIJk_EHmnHyGGAR8Y5jvZPfCyw","name":"Minami-Asagaya","lat":35.69945510000001,"lng":139.635551},{"id":"ChIJNwWo2wqNGGARBaaNSWV-59M","name":"Shin-Otsuka","lat":35.7260747,"lng":139.72983},{"id":"ChIJMwpiebSMGGARPr_454zHvDQ","name":"Yoyogi-Koen","lat":35.671736,"lng":139.6949447},{"id":"ChIJBzXuFB6MGGARDe3XOw7enPQ","name":"Suehirocho","lat":35.702543,"lng":139.771709},{"id":"ChIJEw8CP9-MGGAR6CPmjo-Z_Pk","name":"Higashi-Shinjuku","lat":35.69793,"lng":139.707409},{"id":"ChIJp-KZt8mNGGARZ3K48LYeeIw","name":"Hon-Komagome","lat":35.72504,"lng":139.753654},{"id":"ChIJN-3sWtzyGGAR1GHoyay4Ras","name":"Nakano-Shimbashi","lat":35.6925005,"lng":139.6742859},{"id":"ChIJH_5vQubyGGARBIunuNpUdpI","name":"Nakano-Fujimicho","lat":35.690625,"lng":139.667024},{"id":"ChIJ9U0QBJiOGGARQ1qhN4Ocj6I","name":"Inaricho","lat":35.711375,"lng":139.782208},{"id":"ChIJhZw22riMGGARF95drS3lJ2A","name":"Kita-Sando","lat":35.678905,"lng":139.705105},{"id":"ChIJ8z-tOouLGGARAJ-DluA1t3k","name":"Sakuradamon","lat":35.67729500000001,"lng":139.751379},{"id":"ChIJ11UgDhONGGARWQ6YJVcg4-U","name":"Zoshigaya","lat":35.7188936,"lng":139.718555},{"id":"ChIJE5BUVVWSGGARWQ-JxZ_Zxjw","name":"Shimo","lat":35.7793757,"lng":139.7277106},{"id":"ChIJlcnDp4iNGGARiwvm31U7ZpQ","name":"Nishigahara","lat":35.745926,"lng":139.742238}]
```

Comment:
Make sure you pull from master, and run `yarn test` to check if all the test pass. I think the linters will give you some errors for this code right now.